pkg/k8s/client.go

```diff
@@ -42,7 +42,7 @@ var (
 	k8sCLI = &K8sClient{}
 
 	// k8sWatcherCLI is the client dedicated k8s structure watchers.
-	k8sWatcherCLI = &K8sClient{}
+	k8sWatcherCLI = &K8sSlimClient{}
```

Comment:
The `K8sClient` is for the non-slim objects.

Reply:
> its RESTClient() probably would work for non-slim core objects.

But isn't that problematic because it would fetch the slim version of the core objects?

Reply:
The slim Clientset's `RESTClient()` is the same as the normal k8s one. Only difference to `kubernetes.Interface` for slim is that it overrides the corev1, discoveryV1Beta1, discoveryV1 and networkingV1 typed clients with the slim ones that just unmarshal fewer fields.